wp-security-audit-log/custom-alerts.php

```diff
@@ -143,6 +143,23 @@
                 'yoast-seo-metabox',
                 'enabled',
             ],
+            [
+                8850,
+                WSAL_INFORMATIONAL,
+                __( 'User changed Breadcrumbs Title for a post', 'activity-log-wp-seo' ),
+                __( 'Changed the Breadcrumbs Title for the post %PostTitle% to %new_breadcrumb%.', 'activity-log-wp-seo' ),
```

Comment:
Should some parts of the message be in bold? I am just double checking as it seems to be the direction we took with a lot of other events.

For example:
```suggestion
                __( 'Changed the <strong>Breadcrumbs Title</strong> for the post %PostTitle% to %new_breadcrumb%.', 'activity-log-wp-seo' ),
```